Containerfile

```diff
@@ -1,24 +1,29 @@
 # This file contains the configuration for containerized esi-leap
 
 # Use python as the parent image for esi-leap
-FROM python:3.11
+FROM docker.io/python:3.11
 
 LABEL description="ESI-LEAP -- ESI lease policy manager"
 
+# Indicate that service listens on this port
+EXPOSE 7777
+
 # The working directory for esi-leap inside the container
 WORKDIR /esi-leap
 
-# Copy the files of esi leap inside the container
-COPY run_services.sh run_services.sh
+# Copy the requirements file and install dependencies
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
 
+# Create necessary directories and files
 RUN mkdir -p /var/log/esi-leap
 RUN touch /var/log/esi-leap/esi-leap-dbsync.log
 
-# Run command to install dependencies for esi-leap
-RUN apt update -y && apt install -y python3-pymysql
-RUN pip3 install esi-leap mysql-connector pymysql
```

Comment:
Because you are no longer running `pip install esi-leap`, the resulting image does not actually have `esi-leap` installed. Attempting to run the image results in:

```
./run_services.sh: line 16: esi-leap-dbsync: command not found
```

You should ensure the image runs correctly (before submitting the pull request).

There are two ways of resolving this:

1. You could just re-introduce `pip install esi-leap`, but that is problematic because the version of esi-leap installed in the image won't match the version of esi-leap checked out in the local directory. This means the Containerfile wouldn't be useful for automatically building images on commits to `main`, for example.

2. Update the Containerfile to copy in the contents of the local directory and then `pip install .`.

I strongly favor option 2. @tzumainn?

Reply:
I already made that change in the file I tested. I think during commit I forgot to do git add but the changes are already made and I tested it for sure. I will just push the latest commit. Sorry for this silly mistake.